src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,34 @@
+package bit.javaoop;
+
+public class FixedBudgetManager extends Manager {
+
+    private final int limit;
+
+    public boolean canBeHired(Employee employee){
+        return super.canBeHired() && (getSumOfAllEmployeesSalaries()+employee.getSalary()) < limit;
+    }
+
+    public int getSumOfAllEmployeesSalaries(){
+        int sum = 0;
+        for(int i = 0; i < getCounter(); i++){
+            sum += getTab()[i].getSalary();
+        }
+        return sum;
+    }
+    @Override
+    public boolean isSatisfied(){
+        return getSalary() > 20000 && (limit - getSumOfAllEmployeesSalaries()) < 15000; // to troche nielogiczne, nie powinno byc > 15000?
```

Comment:
- Ja wiem, że to tylko takie nasze tu wewnętrzne zajęcia i to nie ma znaczenia a my piszemy do was review po polsku ale jak już kod jest ładnie napisany po angielsku to fajnie jakby komentarze też były po angielsku :D.

- Co do pytania zamysł był taki że manadżer jest zadowolony jeśli nie ma marnujących się wolnych pieniędzy stąd < 15000.

- Przy użyciu && bądź || warto każdywarunek dodatkowo zanawiasować wygląda to wtedy odrobinę przejrzyściej IDE powinno też pod alt + enter podpowiedzieć to.